WineCellar/WineCellar.GUI/.gitignore

```diff
@@ -9,7 +9,7 @@
 *.user
 *.userosscache
 *.sln.docstates
-
+*.appsettings.json
```

Comment:
Appsettings moet meegenomen worden, dit wordt bij het installeren/builden gekopieerd en gebruikt door de applicatie. Aangezien dit bestand niet optioneel is, zal de applicatie crashen door de afwezigheid van de appsettings.json.

Gebruik voor development "appsettings.Development.json".